src/Raven.Client/Documents/Session/IAdvancedSessionOperations.ConditionalLoad.cs

```diff
@@ -0,0 +1,21 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IDocumentSession.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Raven.Client.Documents.Session
+{
+    /// <summary>
+    ///     Advanced synchronous session operations
+    /// </summary>
+    public partial interface IAdvancedSessionOperations
+    {
+        /// <summary>
+        ///     Conditional load the specified entity with the specified id and changeVector.
```

Comment:
I think we need to explain what is the condition actually. So user will know that if etag matches it won't send document over the wire.